src/renderers/WebGLRenderer.d.ts

```diff
@@ -315,7 +315,7 @@ export class WebGLRenderer implements Renderer {
 
 	renderBufferDirect(
 		camera: Camera,
-		fog: Fog,
+		fog: IFog | null,
```

Comment:
This type declaration is my best guess after reading a bit about interfaces and nullable types. I don't really know TypeScript. On the other hand, it seemed incomplete before.

The change of putting IFog (the fog interface) in a separate d.ts file felt natural, but I don't know if it is correct.

Reply:
OK. @Mugen87 The (too complex) conflicts are with the ts files. I think I am able to fix them, but I feel I am still shooting in the dark with TS (TypeScript, not Tourettes Syndrome, although that would be a funny interpretation). Is there a TS expert around here, or are we all shooting in the dark for the time being? ;-)

Reply:
I would say try your best and then let the commit review by others.